tests/src/test/scala/com/nvidia/spark/rapids/OrcQuerySuite.scala

```diff
@@ -92,4 +97,53 @@ class OrcQuerySuite extends SparkQueryCompareTestSuite {
   ) {
     frame => frame
   }
+
+  private def getColumnEncoding(orcDir: File): (String, String) = {
+    val orcFile = orcDir.listFiles(f => f.getName.endsWith(".orc"))(0)
+    val p = new Path(orcFile.getCanonicalPath)
+    val conf = OrcFile.readerOptions(new Configuration())
+    var reader: Reader = null
+    try {
+      reader = OrcFile.createReader(p, conf)
+      withResource(reader.rows().asInstanceOf[RecordReaderImpl]) { rows =>
+        val stripe: StripeInformation = reader.getStripes.get(0)
+        val stripeFooter = rows.readStripeFooter(stripe)
+        (stripeFooter.getColumns(1).getKind.toString,
+            stripeFooter.getColumns(2).getKind.toString)
+      }
+    } finally {
+      reader match {
+        case c: AutoCloseable => c.close()
```

Comment:
nit: There is an `Arm.withResourceIfAllowed` that does exactly the same thing.

```
// cdh321, cdh330 use a lower ORC version, and the reader is not a AutoCloseable
withResourceIfAllowed(OrcFile.createReader(p, conf)) { reader =>
 withResource(reader.rows.asInstanceOf[RecordReaderImpl]) { rows =>
    ...
  }
}
```

Reply:
Done.